File: app/src/main/java/org/mifos/mobilebanking/ui/activities/base/SplashActivity.java
package org.mifos.mobilebanking.ui.activities.base;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import org.mifos.mobilebanking.ui.activities.LoginActivity;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Intent intent = new Intent(this, LoginActivity.class);
        startActivity(intent);
        finish();
    }
}

File: androidApp/src/main/java/org/mifos/mobile/ui/activities/SplashActivity.kt
package org.mifos.mobile.ui.activities

import android.content.Intent
import android.os.Bundle
import androidx.activity.ComponentActivity

/*
* Created by saksham on 01/June/2018
*/
class SplashActivity : ComponentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        // TODO:: check for user logged in or not, and if logged in move to PasscodeActivity instead.
        val intent = Intent(this, LoginActivity::class.java)

        startActivity(intent)
        finish()
    }
}
